dev/pyproject.py

```diff
@@ -281,6 +361,42 @@ def build(package_type: PackageType) -> None:
         subprocess.check_call([taplo, "fmt", out_path])
 
 
+def _get_package_data(package_type: PackageType) -> Optional[dict[str, list[str]]]:
+    if package_type == PackageType.TRACING:
+        return None
+
+    package_data = {
+        "mlflow": [
+            "store/db_migrations/alembic.ini",
+            "temporary_db_migrations_for_pre_1_users/alembic.ini",
+            "pyspark/ml/log_model_allowlist.txt",
+            "server/auth/basic_auth.ini",
+            "server/auth/db/migrations/alembic.ini",
+            "models/notebook_resources/**/*",
+        ]
+    }
+
+    if package_type != PackageType.SKINNY:
+        package_data["mlflow"] += ["models/container/**/*", "server/js/build/**/*"]
+
+    return package_data
+
+
+def _check_skinny_tracing_mismatch(skinny_reqs: list[str], tracing_reqs: list[str]) -> None:
+    """
+    Check if the tracing requirements are a subset of the skinny requirements.
+    NB: We don't make mlflow-tracing as a hard dependency of mlflow-skinny because
+    it will complicate the package management (need another .release.toml file
+    that is dependent by pyproject.release.toml)
+    """
+    if set(tracing_reqs) - set(skinny_reqs):
+        raise RuntimeError(
+            "Tracing requirements must be a subset of skinny requirements. "
+            "Please check the requirements/skinny-requirements.txt and "
+            "requirements/tracing-requirements.txt files."
+        )
```

Comment:
```suggestion
    if diff := set(tracing_reqs) - set(skinny_reqs):
        raise RuntimeError(
            "Tracing requirements must be a subset of skinny requirements. "
            "Please check the requirements/skinny-requirements.txt and "
            "requirements/tracing-requirements.txt files."
        )
```

let's include diff in the message